.github/workflows/tests-on-packages.yml

```diff
@@ -0,0 +1,65 @@
+name: Run tests on APT packages
+
+on:
+  workflow_call:
+    inputs:
+      pkg_type:
+        required: true
+        type: string
+      revision_name:
+        required: true
+        type: string
+
+jobs:
+  TestPackagesWithLight:
```

Comment:
Just a reminder for myself: the name of the job should be simply: TestPackages or something like this.
Just for the note in my first version I have also added to execute Light for the installed syslog-ng (but it was removed, there was some complexity)

Reply:
fixed